pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -118,6 +118,16 @@ func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
 // ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
 // +k8s:openapi-gen=true
 type ScrapeConfigSpec struct {
+	// The value of the `job` label assigned to the scraped metrics by default.
+	//
+	// Note: This field does not populate the `job_name` field of the generated configuration.
+	// The `job_name` field is always controlled by the operator to prevent duplicate job names,
+	// which Prometheus does not support. Instead, all the metrics from this scrape job are
+	// labelled with the JobName by means of relabeling configs.
```

Comment:
```suggestion
	// The `job_name` field in the rendered scrape configuration is always controlled by the operator to prevent duplicate job names,
	// which Prometheus does not allow. Instead the `job` label is set
	// by means of relabeling configs.
```